docs/en/query_language/agg_functions/reference.md

```diff
@@ -301,6 +300,100 @@ GROUP BY timeslot
 └─────────────────────┴──────────────────────────────────────────────┘
 ```
 
+## skewPop
+
+Computes the [skewness](https://en.wikipedia.org/wiki/Skewness) for sequence.
+
+```
+skewPop( expr )
+```
+
+**Parameters**
+
+`expr` — [Expression](../syntax.md#syntax-expressions) returning a number.
+
+**Returned value**
+
+The skewness of given distribution. Type — [Float64](../../data_types/float.md)
+
+**Example of Use**
+
+```sql
+SELECT skewPop( value ) FROM series_with_value_column
+```
+
+## skewSamp
+
+Computes the [sample skewness](https://en.wikipedia.org/wiki/Skewness) for sequence.
+
+It represents an unbiased estimate of the skewness of a random variable, if the values passed to the function are a sample of this random amount.
```

Comment:
"amount" looks out of place, maybe something like "...if passed values form it's sample"

Reply:
This sentence comes from the description of `varSamp`. It would be good to update that one too.

Reply:
Done.